samples/algorithms/MajoranaQubits/src/Main.qs

```diff
@@ -0,0 +1,17 @@
+/// # Sample
+/// Majorana Qubits
+///
+/// # Description
+/// In hardware providing majorana qubits, common quantum operations
+/// are implemented using measurements and Pauli corrections. This
+/// sample shows a hypotetical hardware provider exposing some custom
+/// gates to Q# and a small library built on top of it.
+
+/// Sample program using gates for custom hardware provider.
+operation Main() : (Result, Result) {
+    use qs = Qubit[2];
+    GateSet.BellPair(qs[0], qs[1]);
+    let res = GateSet.BellMeasurement(qs[0], qs[1]);
+    ResetAll(qs);
+    res
```

Comment:
At the risk of complicating things, I think it could be helpful to show how the Bell measurement correctly distinguishes between each of the four bell states. You could do this by using `BellPair` and `BellMeasurement` to generate the first value, which is always (Zero, Zero), then show that doing an `X` in between to one qubit takes you to a distinct measured Bell state, then a `Z`, then both `X` and `Z`, completing the set of four states.

Reply:
I just tried this, and it worked beautifully. Definitely worth adding, I'm just trying to come up with the right way to include it in the sample. Maybe as commented lines that user can uncomment before running the code again?

Reply:
If this were just a sample for simulation, I would suggest including all of them or even a loop or something, and then returning a list of tuples. But given that this is a sample where we want to encourage the user to generate QIR and see how the custom measurements show up, I think keeping the `Main` smaller is best, so I like your idea of having the commented out lines and some explanation there!

Reply:
I did that for now, but I'm open to other suggestions

Reply:
I did that. Open to suggestions to make those commented lines more appealing to the eye 😄.

Reply:
I think the comments look great!